runtime/doc/develop.txt

```diff
@@ -304,17 +304,18 @@ See also |dev-naming|.
   - accept iterable instead of table
     - exception: in some cases iterable doesn't make sense, e.g. spair() sorts
       the input by definition, so there is no reason for it to accept an
-      iterable, because the input needs to be "hydrated", it can't operate on
+      iterable, because the input needs to be "reified"; it can't operate on
       a "stream".
   - return iterable instead of table
   - mimic the pairs() or ipairs() interface if the function is intended to be
     used in a "for" loop.
-  - when a result-or-error interface is needed, return `result|nil, errmsg|nil`: >
+  - when a result-or-error interface is needed, return `result|nil, nil|errmsg`: >
     ---@return Foo|nil # Result object, or nil if not found.
     ---@return nil|string # Error message on failure, or nil on success.
 <
     - Examples: |vim.ui.open()| |io.open()| |luv-error-handling|
 
+                                                        *dev-patterns*
```

Comment:
Causes CI to fail due to duplicated header of `*dev-patterns*`. The other is on line 473.